tests/analytics.py

```diff
@@ -0,0 +1,499 @@
+    print("analytics.py output_1.json output_2.json section EFI_SECTION_RAW")
+    print("\n")
+    exit()
+    
+# Opening 1st JSON file
+first_file = open(sys.argv[1])
+# Opening 2nd JSON file
+second_file = open(sys.argv[2])
+# returns JSON object as a dictionary
+Json_data_first_file = json.load(first_file)
+Json_data_second_file = json.load(second_file)
+
+    
+
+# search for SectionTypes parameters
+search_key_compression = 'EFI_SECTION_COMPRESSION'
+search_section_key_guid_defined = 'EFI_SECTION_GUID_DEFINED'
+search_section_disposable= 'EFI_SECTION_DISPOSABLE'
+search_section_pe32 = 'EFI_SECTION_PE32'
+search_section_pic = 'EFI_SECTION_PIC'
+search_section_te = 'EFI_SECTION_TE'
+search_section_dxe_depex = 'EFI_SECTION_DXE_DEPEX'
+search_section_version = 'EFI_SECTION_VERSION'
+search_section_user_interface = 'EFI_SECTION_USER_INTERFACE'
+search_section_compatibility16 = 'EFI_SECTION_COMPATIBILITY16'
+search_section_fv_image = 'EFI_SECTION_FIRMWARE_VOLUME_IMAGE'
+search_section_freeform_subtype_guid = 'EFI_SECTION_FREEFORM_SUBTYPE_GUID'
+search_section_raw = 'EFI_SECTION_RAW'
+search_section_pei_depex = 'EFI_SECTION_PEI_DEPEX'
+search_section_mm_depex = 'EFI_SECTION_MM_DEPEX'
+
+#search for FileType parameters
+search_key_filetype_driver = 'FV_FILETYPE_DRIVER'
+search_key_filetype_application = 'FV_FILETYPE_APPLICATION'
+search_key_filetype_freeform = 'FV_FILETYPE_FREEFORM'
+search_key_filetype_peim = 'FV_FILETYPE_PEIM'
+search_key_filetype_smm = 'FV_FILETYPE_SMM'
+search_key_filetype_raw = 'FV_FILETYPE_RAW'
+search_key_filetype_fv = 'FV_FILETYPE_FIRMWARE_VOLUME_IMAGE'
+search_key_filetype_pad = 'FV_FILETYPE_FFS_PAD'
+
+def findkeys(node, kv):
+    if isinstance(node, list):
+        for i in node:
+            for x in findkeys(i, kv):
+               yield x
+    elif isinstance(node, dict):
+        if kv in node:
+            yield node[kv]
+        for j in node.values():
+            for x in findkeys(j, kv):
+                yield x
+
+def plot_piechart_filetype():
+    ''' 
+    function to plot pie chart based on file type
+    '''
+    #fetch the key first
+    list_file_type.append(list(findkeys(Json_data_first_file, 'Type')))
+
+    #fetch the desired values now
+    count_filetype_driver.append(list_file_type[0].count(search_key_filetype_driver))
+    count_filetype_application.append(list_file_type[0].count(search_key_filetype_application))
+    count_filetype_freeform.append(list_file_type[0].count(search_key_filetype_freeform))
+    count_filetype_peim.append(list_file_type[0].count(search_key_filetype_peim))
+    count_filetype_smm.append(list_file_type[0].count(search_key_filetype_smm))
+    count_filetype_raw.append(list_file_type[0].count(search_key_filetype_raw))
+    count_filetype_fv.append(list_file_type[0].count(search_key_filetype_fv))
+ 
+    if count_filetype_driver[0] > 0:
+        print(f'{count_filetype_driver[0]} occurences of {search_key_filetype_driver} in {sys.argv[1]}')
```

Comment:
It's better to use argument parser module instead of manually parsing arguments.

Also if given more preference to Function API... it'll be more scalable solution